lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # These dependencies are only required when certain config options are set
 psycopg2==2.6.1
-WebError==0.11
+WebError==0.10.3
```

Comment:
I think this line can just be removed, since WebError is already in requirements.txt